### PR TITLE
Implement validation functions

### PR DESCRIPTION
**Pull request by Asitha**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: src/strings/Module.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Hello World
+
+Prints "hello world" to command line output
```
Comment (Shakila) [inline src/strings/Module.md]: Don't we need to add samples here?

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,114 @@
+<!--
+  ~ Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (Shakila) [inline pom.xml]: Shall we change the year?

**Review by Shakila — APPROVED** [review]

File: src/strings/tests/main_test.bal [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+# Before Suite Function
+
+string [] ipList = ["172.300.9.28", "172.8.9.28"];
+
+# Test function
+
+@test:Config{}
+function testEmailValidation () {
+    io:println("-----------------Test case for isEmail method------------------");
```
Comment (Nipunich) [inline src/strings/tests/main_test.bal]: Shall we change io:println() to log:info() ?

Comment (Shakila) [reply src/strings/tests/main_test.bal]: Since this is test, we don't have to keep the history in the log file.

File: src/strings/Module.md (hunk 1 above)
Comment (IrushiL) [inline src/strings/Module.md]: Shall we update the Module.md file?

**Review by pramodya1994 — APPROVED** [review]

**Pull request merged**